packages/apps/plugins/plugin-kanban/src/components/KanbanCard.tsx

```diff
@@ -30,13 +36,24 @@ export const KanbanCardComponent: FC<{
   debug?: boolean;
   onDelete?: () => void;
 }> = ({ column, item, debug = false, onDelete }) => {
+  const { themeMode } = useThemeContext();
   const { t } = useTranslation(KANBAN_PLUGIN);
   const { isDragging, attributes, listeners, transform, transition, setNodeRef } = useSortable({
     id: item.id,
     data: { type: 'item', column },
   });
   const tx = transform ? Object.assign(transform, { scaleY: 1 }) : null;
-  const model = useTextModel({ text: item.title });
+
+  const { doc, accessor } = useDocAccessor(item.title);
+  const extensions = useMemo(
+    () => [
+      //
```

Comment:
?

Reply:
removed. but my way of defeating prettier.